.travis.yml

```diff
@@ -37,6 +37,7 @@ jobs:
       name: PyLint
       script:
         - pip install -r requirements.dev.txt
+        - pip install -e .
```

Comment:
Curious why this was added and when it wasn't necessary previously.

Reply:
It's because of the dependency on a yaml library. PyLint wants to import the code, so it needs the import to resolve. Since we now have 1 extra 3rd party dependency, we need to do the install for online linting to pass.

Reply:
This is the sort of information that would be handy in a commit message.

Reply:
Fixed.